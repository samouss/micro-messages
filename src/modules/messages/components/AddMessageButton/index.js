// @flow

import React from 'react';
import Plus from 'components/Plus';
import { CircleButton } from 'components/Button';
import './index.css';

const AddMessageButton = (props: Object) => (
  <div
    styleName="AddMessageButton"
  >
    <CircleButton
      {...props}
    >
      <Plus />
    </CircleButton>
  </div>
);

export default AddMessageButton;
