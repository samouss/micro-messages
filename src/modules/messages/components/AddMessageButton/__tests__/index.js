// @flow

import React from 'react';
import { shallow } from 'enzyme';
import AddMessageButton from '../index';

describe('<AddMessageButton />', () => {
  const defaultProps = {
    onClick: () => {},
  };

  it('expect to render the component correctly', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <AddMessageButton
        {...props}
      />,
    );

    expect(component).toMatchSnapshot();
  });
});
