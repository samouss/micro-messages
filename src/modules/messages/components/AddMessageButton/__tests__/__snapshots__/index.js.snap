// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddMessageButton /> expect to render the component correctly 1`] = `
<div
  styleName="AddMessageButton"
>
  <Circle
    onClick={[Function]}
  >
    <Plus />
  </Circle>
</div>
`;
